.gitignore

```diff
@@ -4,3 +4,5 @@
 /node_modules
 /config/travis/deploy.key
 .*
+*.diff.png
```

Comment:
Don't we want to check in the diffs?

Reply:
the reference images are there and will get overridden when approved, the diffs are generated by each build so they don't need to be checked in